nautobot/extras/tests/integration/test_configcontextschema.py

```diff
@@ -0,0 +1,69 @@
+from nautobot.utilities.testing.integration import SplinterTestCase
+
+
+class ConfigContextSchemaTestCase(SplinterTestCase):
+    """
+    Integration tests for the ConfigContextSchema model
+    """
+
+    def setUp(self):
+        super().setUp()
+        self.user.is_superuser = True
+        self.user.save()
+        self.login(self.user.username, self.password)
+
+    def tearDown(self):
+        self.logout()
+        super().tearDown()
+
+    def test_create_valid_config_context_schema(self):
+        """
+        Given a clean slate, navigate to and fill out the form for a valid schema object
+        Assert the object is successfully created
+        And the user is redirected to the detail page for the new object
+        """
+        # Navigate to ConfigContextSchema list view
+        self.browser.visit(self.live_server_url)
+        self.browser.links.find_by_partial_text("Extensibility").click()
+        self.browser.links.find_by_text("Config Context Schemas").click()
+
+        # Click add add button
+        self.browser.find_by_xpath("/html/body/div/div[1]/a").click()
```

Comment:
I assume you probably copied this using the debug inspector, but seems like it might be fragile if we end up moving things around. Would it be better to find the button by name or button text?"

Reply:
Yeah, this one is a little weird because there are many buttons (links in this case) with the text "Add" and the one we want is in the middle of the list which implies the same indexing fragility. I will play around with the splinter API and see if I can find a better way.